cw/cw305/pyXKCP/__init__.py

```diff
@@ -0,0 +1,5 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+
+from . import pyXKCP_extention_build
```

Comment:
I am not a Python expert but I think the "right" way of doing this would be to have a completely empty `__init__.py` and put the  `from . import pyXKCP_extention_build` line into `cw/cw305/pyXKCP/pyXKCP.py.` Do you think that would work?

Reply:
This should work, however, I am neither a Python expert.

Reply:
I think the best way is either the __init__py in its current version, or require a manual build. What do you think?